constants/megaMenuLinks.ts

```diff
@@ -611,6 +639,27 @@ const QUANTUM_ALGORITHMS_FOR_APPS : MegaDropdownMenuGroup = {
       segment: {
         cta: 'flexible-representation-of-quantum-images-frqi', location: sectionApps
       }
+    },
+    {
+      label: 'Quantum Facial Expression Recognition',
```

Comment:
I believe this should follow the same label shown [here](https://github.com/Qiskit/platypus/blob/73923e76ff1fb8f328e6e51e836e5377528289d2/frontend/vue/components/constants/megaMenuLinks.ts#L643)